docs/source/tracking.rst

```diff
@@ -91,6 +91,10 @@ directory. The URI defaults to ``mlruns``.
 :py:func:`mlflow.create_experiment` creates a new experiment and returns its ID. Runs can be
```

Comment:
I left this around, to enable users to create experiments using non-default `artifact-location` and any other future customizations for experiment creation flow.

Reply:
I'm not sure potential future customizations are a great reason. We initially did plan to remove `create_experiment` since it's anyway accessible in the tracking API (duplication of an identical API is confusing) and because the notion of providing a non-default artifact location is anyway a more advanced feature (I don't think we have any examples of this).

Reply:
There is value to having a clean API to create an experiment with non-default artifact store. The only other way to do this is `mlflow.tracking.fluent.create_experiment` which is super kludge.

Reply:
Er -- it's `mlflow.tracking.MlflowClient().create_experiment`, right?

Reply:
ok! that would work. We would have expose `MlflowClient` in `mlflow/tracking/__init__.py` but that is prolly a good idea anyways.

Reply:
We were initially planning to remove create_experiment, so hopefully @mateiz is still on board with that.